src/Files.App/Views/Settings/AboutPage.xaml

```diff
@@ -160,7 +160,7 @@
 							Margin="10"
 							Background="Transparent"
 							LinkClicked="MarkdownTextBlock_LinkClicked"
-							Text="{x:Bind ViewModel.ThirdPartyNotices, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
+							Text="{x:Bind ViewModel.ThirdPartyNotices, Mode=OneWay}" />
```

Comment:
Avoid unrelated changes.
FYI, When you create AboutPage improvements in another PR, please move methods in AboutPage's code-behind to the ViewModel.

Reply:
Reverted.

> please move methods you've created for 3rd-party stuff in AboutPage's code-behind to the ViewModel.

Which methods?